docs/docs/widgets/statistics.md

```diff
@@ -5,63 +5,65 @@ title: Statistics
 
 # Statistics
 
-**Statistics** can be used to display different statistical information.
+**Statistics** help us understand and summarize data in a clear and simple way. By using tools like graphs, charts, and averages, statistics allow us to see patterns, compare information, and make sense of numbers. Whether it's for research, business, or everyday decisions, statistics turn raw data into useful insights, making complex information easier to understand and act on.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 | <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> |
 |:------------ |:-------------|
 | Primary value label | This property lets you add and remove primary value label. | 
 | Primary value | This property lets you add and remove primary value, the actual amount can be represented here. |
-| Hide secondary value | This property lets you show/hide the secondary value from the statistics component. By default, this property is disabled. Toggle it on to hide the secondary value, you can also click on the `Fx` button next to it to dynamically set the value to `{{true}}` or `{{false}}`. |
+| Hide secondary value | This property lets you show/hide the secondary value from the statistics component. By default, this property is disabled. Toggle it on to hide the secondary value, you can also click on the `fx` button next to it to dynamically set the value to `{{true}}` or `{{false}}`. |
```

Comment:
Hey @baibhavKumar1 the fx should be bold and not enclosed in backticks, can you please make this change in the whole doc and both versions wherever applicable!

Reply:
@RudraPratik30 just in statistics.md or whole doc?

Reply:
Resolved in statistics.md in both version.